packages/cli/src/lib/commands/init/command-impl.ts

```diff
@@ -1,17 +1,16 @@
-import {RcJsonAsArgv} from "../../types";
+import { InitOptions } from './options';
+import { log, logVerbose } from '../../core/loggin';
+import { run } from '../../core/processing/behaviors';
+import { collectRcJson } from './processes/collect-rc-json';
+import { getGlobalOptionsFromArgv } from '../../global/utils';
+import { getInitCommandOptionsFromArgv } from './utils';
+import { updateRcJson } from './processes/update-rc-json';
+import { handleFlowGeneration } from './processes/generate-userflow';
+import { handleGhWorkflowGeneration } from './processes/generate-workflow';
+import { handleBudgetsGeneration } from './processes/generate-lh-budgets';
+import { SETUP_CONFIRM_MESSAGE } from './constants';
 
-import {log, logVerbose} from "../../core/loggin";
-import {run} from "../../core/processing/behaviors";
-import {collectRcJson} from "../init/processes/collect-rc-json";
-import {getGlobalOptionsFromArgv} from "../../global/utils";
-import {getInitCommandOptionsFromArgv} from "./utils";
-import {updateRcJson} from "./processes/update-rc-json";
-import {handleFlowGeneration} from "./processes/generate-userflow";
-import {handleGhWorkflowGeneration} from "./processes/generate-workflow";
-import {handleBudgetsGeneration} from "./processes/generate-lh-budgets";
-import {SETUP_CONFIRM_MESSAGE} from "./constants";
-
-export async function runInitCommand(argv: RcJsonAsArgv) {
+export async function runInitCommand(argv: InitOptions): Promise<void> {
```

Comment:
This i Actually not fully correct, it cannot be properly done yet as we need to fix the middleware first